collections/_article/review-gamescom-2024.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Gamescom 2024"
+excerpt: "Back from gamescom, ready to report."
+categories: ["events"]
+author: Nat
+image: /storage/blog/covers/gamescom-2024.webp
+date: 2024-09-02 18:00:00
+---
+
+Gamescom is more popular than ever. It used to be that you could hide from the masses flooding the entertainment halls by going to the indie area, and playtest one of the few games exhibiting in that much calmer space. Squeezed between the retro, education, and job-market booths, I recall not that many people even knew of its existence - the entire hall was classified as the family-friendly one. 
+Nowadays, the [Indie Arena Booth](https://indiearenabooth.de/gamescom2024/games) alone featured 170 developers, and found competition in multiple other indie groups occupying similar amounts of floor space.
+
+Between these conglomerates and the country/state booths, we managed to count 9 Godot games, some familiar faces as well as a few surprises among them. Regrettably, we are sure to have missed some - if only events would release the exhibitor list with a note on engine technology used. One can only dream… Unless? 
+
+[embed the Twitter thread] 
+
+Since most gamescom demos are still online, here our incomplete list for you to playtest:
+- Cocolation (no demo yet)
+- [Crown Gambit](https://store.steampowered.com/app/2447980/Crown_Gambit/)
+- [Kamaeru: A Frog Refuge](https://store.steampowered.com/app/1978150/Kamaeru_A_Frog_Refuge/)
+- [Kohlrabi Starship](https://store.steampowered.com/app/2337990/Kohlrabi_Starship/)
+- [Koira](https://store.steampowered.com/app/1626620/Koira/)
+- [Oddventure](https://store.steampowered.com/app/1235710/Oddventure/)
+- [PVKK: Planetenverteidigungskanonenkommandant](https://store.steampowered.com/app/2956040/PVKK_Planetenverteidigungskanonenkommandant/)
+- [Spindle](https://store.steampowered.com/app/1386750/Spindle/)
+- [Yarn Guardians](https://store.steampowered.com/app/2928940/Yarn_Guardians/)
+
+*Did you too exhibit at gamescom and want to be added to this list? Email [nat@godotengine.org](mailto:nat@godotengine.org) with a link to your demo.*
```

Comment:
Should we specify "a godot game" to not get random submissions? Though it's probably clear from context already.
```suggestion
*Did you too exhibit a Godot game at gamescom and want to be added to this list? Email [nat@godotengine.org](mailto:nat@godotengine.org) with a link to your demo.*
```